client/src/redux/actions/currentProduct.js

```diff
@@ -0,0 +1,6 @@
+var currentProduct = (product) => ({
```

Comment:
can we name as "setCurrent" or "setCurrentProduct", with type "SET_CURRENT" or "SET_CURRENT_PRODUCT"?

I prefer setCurrent/SET_CURRENT